config.sh

```diff
@@ -84,7 +121,12 @@ function run_tests_in_repo {
 
 EXP_CODECS="jpg jpg_2000 libtiff zlib"
 EXP_MODULES="freetype2 littlecms2 pil tkinter webp"
-EXP_FEATURES="transp_webp webp_anim webp_mux xcb"
+if [ -z "$IS_OSX" ] && [[ "$MB_PYTHON_VERSION" != pypy3* ]]; then
+  EXP_FEATURES="fribidi harfbuzz raqm transp_webp webp_anim webp_mux xcb"
+else
+  # can't find FriBiDi
+  EXP_FEATURES="transp_webp webp_anim webp_mux xcb"
```

Comment:
macOS and PyPy on Ubuntu don't find FriBiDi preinstalled. Adding `brew install fribidi` only finds it on Python 3.6. I'm not sure why, but it should be irrelevant given the `build_ext` flags defined above.